prow/cmd/entrypoint/BUILD.bazel

```diff
@@ -14,6 +14,26 @@ go_library(
     ],
 )
 
+load("@io_bazel_rules_docker//go:image.bzl", "go_image")
+
+container_image(
+    name = "base-image",
+    # TODO: this base image is *different* to the one defined in the Dockerfile?
+    base = "@git-base//image",
+    directory = "/",
```

Comment:
Is it important that the `entrypoint` binary is located at `/`? If so, I will revert changes to these pod utilities as I think this may be a bit tricky and out-of-scope.

Reply:
yeah, unfortunately for every podutil binary they must be at `/`, not just in $PATH